tools/tidb-binlog-cluster.md

```diff
@@ -3,9 +3,13 @@ title: TiDB-Binlog cluster 版本用户文档
 category: advanced
 ---
 
+# TiDB-Binlog 用户文档
+
+本文档介绍 cluster 版本 TiDB-Binlog 的架构以及部署方案。如需使用 local 版本或者 kafka 版本，可参考 [local 版本的 TiDB-Binlog 部署文档](../tools/tidb-binlog.md) 和 [kafka 版本的 TiDB-Binlog 部署文档](../tools/tidb-binlog-kafka.md)。
+
 ## TiDB-Binlog 简介
 
-TiDB-Binlog 是一个用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+TiDB-Binlog 是一个用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。如需部署 local 版本的 TiDB-Binlog，可参考 [local 版本的 TiDB-Binlog 部署文档](../tools/tidb-binlog.md)。
```

Comment:
The meaning of the last sentence is already involved in line 8. I think we can remove it.